src/gt4py/testing/suites.py

```diff
@@ -206,6 +209,23 @@ def hyp_wrapper(test_hyp, hypothesis_data):
     def __new__(cls, cls_name, bases, cls_dict):
         if cls_dict.get("_skip_", False):  # skip metaclass magic
             return super().__new__(cls, cls_name, bases, cls_dict)
+
+        # Grab members inherited from base class
+        required_members = (
+            "domain_range",
+            "symbols",
+            "definition",
+            "validation",
+            "backends",
+            "dtypes",
+        )
+        inherited_members = {
+            key: getattr(bases[0], key)
+            for key in required_members
+            if hasattr(bases[0], key) and key not in cls_dict
+        }
+        cls_dict.update(inherited_members)
```

Comment:
Maybe a bit pedantic, but if it is accepted to go to the base class to look for missing members, shouldn't this approach be general and check all the base class instead of only the first one? Something like this (untested code):
```suggestion
        required_members = (
            "domain_range",
            "symbols",
            "definition",
            "validation",
            "backends",
            "dtypes",
        )
        missing_members = set(cls_dict.keys(0)) - required_members
        inherited_members = {}
        for key in missing_members:
            for base in bases:
                if hasattr(key, base):
                    inherited_members[key] = getattr(base, key) 

        cls_dict.update(inherited_members)
        
```

Reply:
Makes totally sense.